config/routes.rb

```diff
@@ -47,6 +47,7 @@
     resources :sections
     resources :submissions
     resources :topics
+    resources :categories
```

Comment:
I just noticed the order of resources in `routes.rb` is the order they are shown in the admin side-bar. Let's keep it in alphabetical order.